src/components/AddTodo.jsx

```diff
@@ -2,7 +2,7 @@ import { useState, useEffect } from "react";
 import axios from "axios";
 import { useNavigate } from "react-router-dom";
 
-const AddTodo = (props = (todos, setTodos)) => {
+const AddTodo = (todos, setTodos) => {
```

Comment:
You are not destructuring. setTodos even if you pass it to the component will be undefined.

Reply:
I changed it and I think i am doing it properly now